tests/foreman/ui/test_rhc.py

```diff
@@ -20,9 +20,10 @@
 from fauxfactory import gen_string
 from wait_for import wait_for
 
-from robottelo import constants
+from robottelo.api.utils import enable_sync_redhat_repo
 from robottelo.config import settings
 from robottelo.constants import DEFAULT_SUBSCRIPTION_NAME
+from robottelo.constants import REPOS
```

Comment:
Why import explicitly when you can use constants.REPOS and constants.DEFAULT_SUBSCRIPTION_NAME?

Reply:
no reason. want me to change it?

Reply:
yes, please

Reply:
updated the pr.